ogr/services/github.py

```diff
@@ -103,31 +104,55 @@ def __init__(
         read_only=False,
         github_app_id: str = None,
         github_app_private_key: str = None,
+        github_app_private_key_path: str = None,
         **_,
     ):
         super().__init__()
         self.token = token
 
         # Authentication via GitHub app
         self.github_app_id = github_app_id
-        self.github_app_private_key = github_app_private_key
+        self._github_app_private_key = github_app_private_key
+        self.github_app_private_key_path = github_app_private_key_path
 
         self.github = github.Github(login_or_token=self.token)
         self.read_only = read_only
 
+    @property
+    def github_app_private_key(self):
+        if self._github_app_private_key:
+            return self._github_app_private_key
+
+        if self.github_app_private_key_path:
+            if not Path(self.github_app_private_key_path).is_file():
+                raise GithubAPIException(
+                    f"Path to the github-app private key "
```

Comment:
```suggestion
                    f"File with the github-app private key "
```

'Path does not exist' doesn't make sense ;)